nautobot/core/templates/components/panel/footer_table.html

```diff
@@ -0,0 +1,17 @@
+{% block footer_left %}
```

Comment:
Let's for now not add the `footer_left` and `footer_right` blocks until/unless we find we need them.